test/utils/test_draw.py

```diff
@@ -0,0 +1,137 @@
+import pytest
+
+import torch
+import kornia.testing as utils
+from kornia.utils import rectangle
+
+
+class TestRectangle:
+    @pytest.mark.parametrize('batch', (4, 17))
+    @pytest.mark.parametrize('color', (1.0, 0.5))
+    def test_smoke(self, device, batch, color):
+        black_image = torch.zeros(batch, 1, 3, 3, device=device)  # 1 channel 3x3 black_image
+        points = torch.tensor([1., 1., 1., 1.]).to(device).expand(batch, 1, 4)  # single pixel rectangle
```

Comment:
it's recommended to initialize tensor as `torch.tensor(..., device=device, dtype=dtype)`